crs_msgs/srv/LocalizeToPart.srv

```diff
@@ -0,0 +1,5 @@
+sensor_msgs/PointCloud2[] point_clouds
```

Comment:
There should be a comment here that the point clouds need to be in the same coordinate frame, the coordinate frame should be stationary, world,  robot base, etc but not the camera which moves with the robot

Reply:
Noted